techradar/radars/factories.py

```diff
@@ -0,0 +1,29 @@
+import factory.fuzzy
+from factory.django import DjangoModelFactory
+from faker import Faker
+from .models import Radar, Segment, Technology
+
+fake = Faker()
+
+
+class RadarFactory(DjangoModelFactory):
+    label = factory.LazyFunction(fake.word)
```

Comment:
Wygodniejsze może być używanie fakera w taki sposób:

```suggestion
    label = factory.Faker("word")
```

nie musisz wtedy robić na górze `fake = Faker()` :slightly_smiling_face: